### ISSUE TITLE
Not supporting flutter 2.0 (depend on path_provider old version) 

### ISSUE THREAD

**Issue by aswanthtpc** [sbis04/video_trimmer#97]
When i try upgrade my flutter version to 2.0 it shows dependency error on path_provider

**Comment by lvlrSajjad** [comment]
https://github.com/sbis04/video_trimmer/pull/98
I have upgraded packages in this pull request. :-D

**Comment by aswanthtpc** [comment]
is it available with the same version (0.3.5)?

**Comment by aswanthtpc** [comment]
> #98
> I have upgraded packages in this pull request. :-D

is it available with the same version (0.3.5)?

**Comment by lvlrSajjad** [comment]
> > #98
> > I have upgraded packages in this pull request. :-D
> 
> is it available with the same version (0.3.5)?

For now, you can use it this way
```
  video_trimmer:
    git:
      url: https://github.com/lvlrSajjad/video_trimmer.git
      ref: master
```
And if there is a problem tell me to fix it :-D (Because for now, it's working for me)

**Comment by frankdevryer** [comment]
> > > #98
> > > I have upgraded packages in this pull request. :-D
> > 
> > 
> > is it available with the same version (0.3.5)?
> 
> For now, you can use it this way
> 
> ```
>   video_trimmer:
>     git:
>       url: https://github.com/lvlrSajjad/video_trimmer.git
>       ref: master
> ```
> 
> And if you faced a problem tell me to fix it :-D (Because for now, it's working for me)

This works! Great.

**Comment by khanna007** [comment]
@lvlrSajjad This works but when we are going to make apk or IPA its showing error
